docs/source/cn/pooling.py

```diff
@@ -44,3 +44,244 @@
         - **output_size** (Int64List[3]): 目标输出大小（单个整数或包含三个整数的元组）
     """,
 )
+
+reset_docstr(
+    oneflow.nn.AdaptiveAvgPool1d,
+    r"""AdaptiveAvgPool1d(output_size)
+
+    在由多个平面组成的输入信号 `input` 上应用 1D 自适应平均池化。
+
+    对于任何大小的输入，输出大小都是 H 。
+    输出的数量等于输入平面的数量。
+
+    参数：
+        - **output_size** (Int64List[1]): 目标输出大小（单个整数）
+    
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> import oneflow.nn as nn
+
+        >>> m = nn.AdaptiveAvgPool1d(5)
+        >>> input = flow.randn(1, 64, 8)
+        >>> output = m(input)
+        >>> output.size()
+        oneflow.Size([1, 64, 5])
+
+    """
+)
+
+reset_docstr(
+    oneflow.nn.AdaptiveAvgPool2d,
+    r"""AdaptiveAvgPool2d(output_size)
+    
+    在由多个平面组成的的信号 `input` 上应用 2D 自适应平均池化。
+
+    对于任何大小的输入，输出大小都是 H x W 。
+    输出的数量等于输入平面的数量。
```

Comment:
```suggestion
    在由多个平面组成的的信号 `input` 上应用 2D 自适应平均池化。

    对于任何大小的输入，输出大小都是 H x W。
    
    输出的数量等于输入平面的数量。
```
如果中间不空行，渲染的效果就多了个空格，很别扭，所以我们就都加上空行吧~